index.js

```diff
@@ -18,7 +20,10 @@ new Cli({
         callback(reg);
     },
     run: function(port, config) {
-        console.log("Matrix-side listening on port %s", port);
+        log.info("Matrix-side listening on port %s", port);
+        if (config.logging) {
```

Comment:
I'm not sure how the config parsing works in node, but this seems to be looking for something called ".logging", which isn't the ".config" we'd be getting from the sample config above?

Reply:
Ah, this was a mistake. Should have been ``logging`` in the config file.